### ISSUE TITLE
Shared cursors in subplots

### ISSUE THREAD

**Issue by TheDan64** [igiagkiozis/plotly#108]
Is it possible / would it be possible to have a shared cursor across multiple graphs in a subplot? For example in some applications when you have two graphs with related data / x axis (ie time) the cursor would show up on both graphs when you hover over one (usually with a thin vertical bar through the graph)?

**Comment by mfreeborn** [comment]
Yes, I believe that this is supported in the plotly.js library and therefore should be supported in this wrapper library. Here's a related thread from plotly.py but the concept should map across pretty similarly.

**Comment by mfreeborn** [comment]
Closing this pending a minimal working example of plotly.rs behaviour differing from plotly.js.
